proto/astarteplatform/msghub/astarte_message.proto

```diff
@@ -41,14 +41,12 @@ message MessageHubEvent{
 
 /* Astarte message to be used when sending data to Astarte. */
 message AstarteMessage{
-  string interface_name = 1;               // Name of the interface to send data on.
-  string path = 2;                         // Endpoint to send the data on.
-  oneof payload {                          // Content of the message.
-    AstarteDataType astarte_data = 3;      // Effective payload.
-    AstarteUnset astarte_unset = 4;        // Null payload.
+  string interface_name = 1;                        // Name of the interface to send data on.
+  string path = 2;                                  // Endpoint to send the data on.
+  optional google.protobuf.Timestamp timestamp = 3; // Explicit timestamp for the message transmission.
+  oneof payload {                                   // Content of the message.
+    AstarteDatastreamInidividual datstream_individual = 4; // An individual data type.
+    AstarteDatastreamObject datstream_object = 5;          // An object data type.
```

Comment:
```suggestion
    AstarteDatastreamInidividual datastream_individual = 4; // An individual data type.
    AstarteDatastreamObject datastream_object = 5;          // An object data type.
```

Reply:
I notice this because the enum in rust gets generated with the name of this variable